Header header
uint32 arm_clamp
uint32 arm_roll
uint32 arm_elbow
uint32 arm_shoulder
uint32 arm_yaw
uint32 time_to_complete
